mimic/model/clb_objects.py

```diff
@@ -260,6 +260,42 @@ def add_node(self, node_list, lb_id, current_timestamp):
 
         return not_found_response("loadbalancer"), 404
 
+    def del_load_balancer(self, lb_id, current_timestamp):
+        """
+        Returns response for a load balancer
```

Comment:
The wrapping here looks a little off... I should probably give you my python-docstring-wrapping script.